<launch>
	<node name="interface_kit" pkg="phidgets_ros" type="interface_kit.py" output="screen">
	    <!-- <param name="name" type="str" value="my_interface_kit"/> -->
		<param name="serial_number" type="int" value="97608"/>
	</node>
</launch>